tpu/src/test/java/tpu/QueuedResourceIT.java

```diff
@@ -120,6 +124,33 @@ public void testGetQueuedResource() throws IOException {
     }
   }
 
+  @Test
+  public void testListTpuVm() throws IOException {
+    try (MockedStatic<TpuClient> mockedTpuClient = mockStatic(TpuClient.class)) {
+      QueuedResource queuedResource1 = mock(QueuedResource.class);
+      QueuedResource queuedResource2 = mock(QueuedResource.class);
+      List<QueuedResource> mockListQueuedResources =
+          Arrays.asList(queuedResource1, queuedResource2);
+
+      TpuClient mockClient = mock(TpuClient.class);
+      mockedTpuClient.when(TpuClient::create).thenReturn(mockClient);
+      TpuClient.ListQueuedResourcesPagedResponse mockListQueuedResourcesResponse =
```

Comment:
Same comment here. Import the class and simplify the data type.

Reply:
Thank you1 Fixed